### PR TITLE
QAART-451 Address Sonar's Major issue: Field names should comply with a ...

### PR DESCRIPTION
**Pull request by rochan**
...naming convention

The following code snippet illustrates this rule when the regular expression value is "^[a-z][a-zA-Z0-9]*$":

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/visualeditordialogs/VisualEditorHyperLinkDialog.java [hunk 1]
```diff
@@ -42,17 +42,17 @@
...
 
 	private int[] pageCategoryIndex = new int[4];
-	private final int NEWPAGEINDEX = 0;
-	private final int MATCHINGPAGEINDEX = 1;
-	private final int EXTERNALLINKINDEX = 2;
-	private final int REDIRECTPAGEINDEX = 3;
+	private final int newPageIndex = 0;
+	private final int matchingPageIndex = 1;
+	private final int externalLinkIndex = 2;
+	private final int redirectPageIndex = 3;
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/visualeditordialogs/VisualEditorHyperLinkDialog.java]: could be  static

Comment (rochan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/visualeditordialogs/VisualEditorHyperLinkDialog.java]: Ok changed the variables to be private static final

**Comment by rochan** [comment]
Apparently this ticket would undo some of the work needed for QAART-452, which is to make private constant private static final.
This PR now take cares of some of the work in QAART-452 as well.

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**